### DISCUSSION TITLE
XFEM Paraview Plugin installation

### DISCUSSION THREAD

**Discussion by rksin8** [idaholab/moose#16426] [Q&A Tools] [answered]
Hi,
Do we still need to install the "XFEM Paraview Plugin" manually along with Paraview source code? As per [discussion here](https://gitlab.kitware.com/paraview/paraview/-/issues/18118), it seems the plugin is now part of ParaView.

Thanks,

Regards,
Ranjeet

**Comment by jiangwen84** [comment, marked as answer]
Yes, you are correct. The plugin is part of Paraview, but you need to Load MooseXfemClip plugin in the Plugin Manager.

**Reply by rksin8** [reply]
Thank you for your reply!  Successfully loaded MooseXfemClip plugin.
